packages/services/src/workers/notification-scanner.js

```diff
@@ -0,0 +1,33 @@
+import { User, UserNotification, UserNotificationType } from '@aragonone/court-backend-server/build/models/objection'
+import * as notificationScanners from '../models/notification-scanners'
+
+/**
+ * This worker loops over all notification scanner objects and 
+ * inserts a notification DB entry for every email that should be sent
+ */
+export default async function (logger) {
+  Object.keys(notificationScanners).forEach(async (model) => {
+    await tryRunScanner(logger, model)
+  })
+}
+
+export async function tryRunScanner(logger, model) {
+  const type = await UserNotificationType.findOrInsert({model})
+  const scanner = notificationScanners[model]
+  const { scannedAt } = type
+  const { scanPeriod } = scanner
+  if (scannedAt && scannedAt.getTime()+scanPeriod > Date.now()) return
```

Comment:
This is something we could abstract in the scanner, sth like `scanner.shouldScanNow()`